### PR TITLE
Document Sentinel fields in INFO command

### LINKED ISSUE #2283: INFO Sentinel not documented
for the info command there is currently no documentation about the sentinel section

### PR DESCRIPTION
**Pull request by enjoy-binbin**
Fixes #2283

### CODE REVIEW

File: commands/info.md [hunk 1]
```diff
@@ -409,6 +410,15 @@ For each error type, the following line is added:
...
 *   `errorstat_XXX`: `count=XXX`
 
+Here is the meaning of all fields in the **sentinel** section:
+
+*   `sentinel_masters`: Number of master monitored by this sentinel.
+*   `sentinel_tilt`: A value of 1 means this sentinel is in TILT mode.
+*   `sentinel_tilt_since_seconds`: Indicates how many seconds this sentinel already is in the TILT mode. If it is not in TILT mode, the value will be -1. Added in Redis 7.0.
+*   `sentinel_running_scripts`: The number of scripts this sentinel is currently executing.
+*   `sentinel_scripts_queue_length`: The length of the queue of user scripts to execute.
+*   `sentinel_simulate_failure_flags`: Flags for the SENTINEL `SIMULATE-FAILURE` command, for testing purposes.
```
Comment (itamarhaber) [inline commands/info.md:420]:
```suggestion
The **sentinel** section is only available in Redis Sentinel instances. It consists of the following fields:

*   `sentinel_masters`: Number of Redis masters monitored by this Sentinel instance
*   `sentinel_tilt`: A value of 1 means this sentinel is in TILT mode
*   `sentinel_tilt_since_seconds`: Duration in seconds of current TILT, or -1 if not TILTed. Added in Redis 7.0.
*   `sentinel_running_scripts`: The number of scripts this Sentinel is currently executing
*   `sentinel_scripts_queue_length`: The length of the queue of user scripts that are pending execution
*   `sentinel_simulate_failure_flags`: Flags for the `SENTINEL SIMULATE-FAILURE` command
```

**Review by itamarhaber — COMMENTED** [review]
:bow: @enjoy-binbin

File: commands/info.md (hunk 1 above)
Comment (enjoy-binbin) [reply commands/info.md]: thanks for the review and the suggestions!

File: commands/info.md [hunk 2]
```diff
@@ -409,6 +410,15 @@ For each error type, the following line is added:
 
 *   `errorstat_XXX`: `count=XXX`
 
+The **sentinel** section is only available in Redis Sentinel instances. It consists of the following fields:
+
+*   `sentinel_masters`: Number of Redis masters monitored by this Sentinel instance
+*   `sentinel_tilt`: A value of 1 means this sentinel is in TILT mode
+*   `sentinel_tilt_since_seconds`: Duration in seconds of current TILT, or -1 if not TILTed. Added in Redis 7.0.
```
Comment (enjoy-binbin) [inline commands/info.md:417]:
```suggestion
*   `sentinel_tilt_since_seconds`: Duration in seconds of current TILT, or -1 if not TILTed. Added in Redis 7.0.0
```

**Pull request merged**